docs/install/operator/configuring-serving-cr.md

```diff
@@ -0,0 +1,393 @@
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  cluster-local-gateway:
+    selector:
+      custom: custom-local-gateway
+  config:
+    istio:
+      local-gateway.knative-serving.cluster-local-gateway: "custom-local-gateway.istio-system.svc.cluster.local"
+```
+
+## High availability:
+
+By default, Knative Serving runs a single instance of each controller. The `spec.high-availability` field allows you to configure the number of replicas for the following master-elected controllers: `controller`, `autoscaler-hpa`, `networking-istio`. This field also configures the `HorizontalPodAutoscaler` resources for the data plane (`activator`):
+
+The following configuration specifies a replica count of 3 for the controllers and a minimum of 3 activators (which may scale higher if needed):
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  high-availability:
+    replicas: 3
+```
+
+## Managing resources for containers
+
+The operator custom resource allows you allows you to configure resource requests and limits for the Knative system containers.
+Requests and limits may be configured for the following containers: `activator`, `autoscaler`, `controller`, `webhook`, `autoscaler-hpa`,
+`networking-istio` and `queue-proxy`.
+
+To override resource settings for a specific container, create an entry in the `spec.resources` list with the container name and the [Kubernetes resource settings](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#resource-requests-and-limits-of-pod-and-container).
+
+For example, the following KnativeServing resource configures the `activator` to request 0.3 CPU and 100MB of RAM, and sets hard limits of 1 CPU, 250MB RAM, and 4GB of local storage:
+
+\```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  resources:
+  - container: activator
+    requests:
+      cpu: 300m
+      memory: 100Mi
+    limits:
+      cpu: 1000m
+      memory: 250Mi
+      ephemeral-storage: 4Gi
+\```
+
+If you would like to add another container `autoscaler` with the same configuration, you need to change your CR as below:
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeServing
+metadata:
+  name: knative-serving
+  namespace: knative-serving
+spec:
+  resources:
+  - container: activator
+      requests:
+        cpu: 30m
+        memory: 40Mi
+        storage: 1Gi
+        ephemeral-storage: 2Gi
+      limits:
+        cpu: 300m
+        memory: 400Mi
+        storage: 2Gi
+        ephemeral-storage: 4Gi
+  - container: autoscaler
+      requests:
+        cpu: 30m
+        memory: 40Mi
+        storage: 1Gi
+        ephemeral-storage: 2Gi
+      limits:
+        cpu: 300m
+        memory: 400Mi
+        storage: 2Gi
+        ephemeral-storage: 4Gi
```

Comment:
The YAML indentation here is off; I also made the quantities in this example align with the one above.

```suggestion
  - container: activator
    requests:
      cpu: 300m
      memory: 100Mi
    limits:
      cpu: 1000m
      memory: 250Mi
      ephemeral-storage: 4Gi
  - container: autoscaler
    requests:
      cpu: 300m
      memory: 100Mi
    limits:
      cpu: 1000m
      memory: 250Mi
      ephemeral-storage: 4Gi
```